jpo-ode-svcs/src/main/java/us/dot/its/jpo/ode/traveler/TravelerMessageBuilder.java

```diff
@@ -233,4 +252,41 @@
 //        }
 //        return area;
 //    }
-//}
+
+    private long getStartTime(TravelerInputData.DataFrame dataFrame) throws ParseException {
```

Comment:
Method should use DateTimeUtils and ZonedDateTime java classes for parsing ISO date-time.